src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalExternalUserFunctionalForV2ApiTest.java

```diff
@@ -116,7 +117,7 @@ public void findOrgByPfmShouldBeSuccess() {
                 professionalApiClient.getMultipleAuthHeaders(pfmBearerToken));
         assertThat(response.get("paymentAccount")).asList().hasSize(3);
         assertThat(response.get("pendingPaymentAccount")).asList().hasSize(0);
-        assertThat(response.get("orgType")).isEqualTo("Doctor");
+        assertThat(response.get("orgType")).isEqualTo(OrganisationTypeConstants.SOLICITOR_ORG);
```

Comment:
I know this is very minor, but I prefer to use Strings in the Tests rather than constants, because If someone accidentally changes that constant, this test will fail

Reply:
That's fair. I'll make that change for the tests